packages/legend-application-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -109,14 +111,30 @@ import {
   property_setName,
   property_setGenericType,
   property_setMultiplicity,
+  class_arrangeProperty,
+  class_arrangeDerivedProperty,
+  class_arrangeConstraint,
+  class_arrangeSuperTypes,
   setGenericTypeReferenceValue,
 } from '../../../../stores/graphModifier/DomainGraphModifierHelper.js';
 import {
   CLASS_PROPERTY_TYPE,
   getClassPropertyType,
 } from '../../../../stores/shared/ModelUtil.js';
 import { LEGEND_STUDIO_APPLICATION_NAVIGATION_CONTEXT_KEY } from '../../../../stores/LegendStudioApplicationNavigationContext.js';
-import { TaggedValueEditor } from './TaggedValueEditor.js';
+import { getEmptyImage } from 'react-dnd-html5-backend';
+
+class ClassPropertyDragSource {
```

Comment:
I wonder if we need any of these classes at all, seems like you don't even construct them, so let's just use `type` instead of `class`. For example:

```ts
type ClassPropertyDragSource = {
  property: Property;
}
```

Reply:
! you're right, switching to type 👍